pkg/tasks/thanos_ruler_user_workload.go

```diff
@@ -91,6 +91,16 @@ func (t *ThanosRulerUserWorkloadTask) create() error {
 		return errors.Wrap(err, "reconciling ThanosRuler ClusterRoleBinding failed")
 	}
 
+	moncrb, err := t.factory.ThanosRulerMonitoringClusterRoleBinding()
+	if err != nil {
+		return errors.Wrap(err, "initializing ThanosRuler monitoring ClusterRoleBinding failed")
```

Comment:
nit, i think we used `Thanos Ruler` elsewhere.

Reply:
Seems like it was mixed, adjust all of them.